pkgs/applications/networking/localproxy/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  openssl,
+  protobuf3_19,
+  catch2,
+  boost181,
+  icu,
+}: let
+  boost = pkgs.boost181.override {enableStatic = true;};
+in
+  stdenv.mkDerivation (finalAttrs: {
+    pname = "localproxy";
+    version = "3.1.0";
+
+    # causes redefinition of _FORTIFY_SOURCE
+    hardeningDisable = ["fortify3"];
```

Comment:
```suggestion
```

Reply:
I see that this comment has been marked as resolved, but nothing has changed in your file.

Can you explain why this is needed at least?

Reply:
I thought he just suggested spaces to go between the "'s and ['s ["fortify3"], which I did.
It's needed like in many other places of nixpkgs (git grep _FORTIFY_SOURCE), without this you would get compiler errors.

Reply:
Thanks.